### PR TITLE
LPS-137753 Create Condition Node

### PR DESCRIPTION
**Pull request by ramonmsp**
Story: [LPS-135890](https://issues.liferay.com/browse/LPS-135890)

### CODE REVIEW

File: modules/apps/portal-workflow/portal-workflow-instance-tracker-web/src/main/resources/META-INF/resources/js/components/nodes/hexagon-nodes/ConditionNode.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import ClayIcon from '@clayui/icon';
+import React from 'react';
+
+import HexagonNode from './HexagonNode';
+
+export default function JoinNode({data: {done = false, label}}) {
```
Comment (naraandrade) [inline modules/apps/portal-workflow/portal-workflow-instance-tracker-web/src/main/resources/META-INF/resources/js/components/nodes/hexagon-nodes/ConditionNode.js]: @ramonmsp  I think you forgot to rename the function to ConditionNode

**Comment by naraandrade** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by ramonmsp** [comment]
ci:test:sf

**Comment by ramonmsp** [comment]
ci:test:relevant

**Review by naraandrade — APPROVED** [review]

**Review by rafaprax — APPROVED** [review]

**Comment by jotascript** [comment]
Just started reviewing :)

**Review by jotascript — APPROVED** [review]

**Comment by ramonmsp** [comment]
ci:test:relevant

**Comment by ramonmsp** [comment]
ci:test:relevant

**Comment by rafaprax** [comment]
Hey @rodrigocunhaa, can you please forward this manually to BChan? Thx

**Comment by rodrigocunhaa** [comment]
Just started reviewing :)

**Comment by rodrigocunhaa** [comment]
Sent here: https://github.com/brianchandotcom/liferay-portal/pull/106567

**Comment by naraandrade** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**